### PR TITLE
[Changelog] Update for Sprint 118

### PR DESCRIPTION
**Pull request by JPrause**
[skip ci]

### CODE REVIEW

**Comment by JPrause** [comment]
@miq-bot add_label documentation

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 
-## Ivanchuk-Beta2
+## Unreleased as of Sprint 118 - 2019-08-19
+
+### Fixed
```
Comment (simaishi) [inline CHANGELOG.md]: I think this should be `Added`.

Comment (JPrause) [reply CHANGELOG.md]: hmmm,...that PR is labeled as a bug (Fixed). But I can change it to Added.

Comment (simaishi) [reply CHANGELOG.md]: Already discussed with @JPrause but for future reference... Adding those modules was a part of a bug fix. But it just looked strange to put that PR in `Fixed` section when there is no "issue" mentioned and title says "Install modules"...

**Pull request merged**